Documentation/Getting-Started-For-Windows.md

```diff
@@ -15,6 +15,7 @@ may be replaced in all commands below with the directory of your choice.
 * Clone and build CoreCLR:
     ```
     $ git clone https://github.com/dotnet/coreclr
+    $ git clone https://github.com/dotnet/corefx
```

Comment:
@adiaaida @alxest did we get to the bottom of this?  I'm wondering why @alxest needed to add corefx.